addons/bestja_frontend_fpbz/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+{
+    # Theme information
+    'name': "FPBŻ frontend",
+    'description': """
+    This is custom theme made for Federacja Polskich Banków Żywności
+    """,
+    'category': 'Theme',
+    'version': '1.0',
+    'css': ['static/src/css/custom.css'],
+    'depends': ['website'],
+
+    # templates, pages, and snippets
+    'data': [
+        'views/config.xml',
+        'views/pages.xml',
+        'views/snippets.xml',
+        'data/images.xml',
+    ],
+    'application': True,
+    # Your information
+    'author': "Laboratorium EE, Kamil Woźniak",
+    'website': "http://laboratorium.ee",
+}
```

Comment:
This red sign indicates that there is no newline at end of file. It is good practice to put such newlines (however it is not critically important).